_std/defines/shuttle.dm

```diff
@@ -15,3 +15,13 @@
 #define SHUTTLETRANSITTIME (2 MINUTES / (1 SECOND))
 
 // you might be asking "why in seconds?" the answer is that shuttle code uses seconds as a base unit and I'm too tired to refactor it
+
+
+#define SHUTTLE_COGMAP   "cogmap"
+#define SHUTTLE_COGMAP2  "cogmap2"
+#define SHUTTLE_DONUT2   "donut2"
+#define SHUTTLE_DONUT3   "donut3"
+#define SHUTTLE_OSHAN    "oshan"
+#define SHUTTLE_MANTA    "manta"
+#define SHUTTLE_DESTINY  "destiny"
+#define SHUTTLE_NODEF    "nodef"
```

Comment:
why are these not directions + special shuttles

Reply:
part of the issue was that directions can't be unique- four directions, seven shuttles (plus `SHUTTLE_NODEF` as a fallback)

Reply:
why don't you have the direction shuttles as directions and the weird ones as the maps

Reply:
Which is which? Right now I'm using these to index into the landmark associative list.